src/js/util/cookieFunctions.ts

```diff
@@ -0,0 +1,57 @@
+import { CookieOptions } from '../types/cookies';
+
+export const getCookies = (cname: string) => {
```

Comment:
Persoonlijk zou ik er voor kiezen om https://github.com/js-cookie/js-cookie te gebruiken. Ja het is een extra dependency maar met 20k stars en 511k usage gaat dit niet zomaar weg. 

Cookies in javascript zijn gewoon niet goed geregeld. Eigenlijk zou je de voorkeur moeten geven aan localStorage maar je wil deze setting eigenlijk ook een expire date kunnen geven en volgens mij is het ook wel gebruikelijk om dit in de cookies op te slaan.

Deze code gaat niet meer over de library die je maakt maar over een low level api en dat wil je naar mijn idee niet